src/components/TutorialMetadata.tsx

```diff
@@ -38,6 +38,7 @@ const PillContainer = styled.div`
 const SkillPill = styled(Pill)`
   align-self: flex-start;
   margin-bottom: 0.5rem;
+  white-space: nowrap;
```

Comment:
Added this because Chinese characters were wrapping every character. Open to suggestions, since this may cause issues in other languages if these terms are longer.